assets/db/articleList.php

```diff
@@ -0,0 +1,16 @@
+<?php 
+(array) $json_articles = file_get_contents("assets/db/db_articles.json");
+(array) $parsed_articles = json_decode($json_articles, true);
+
+// var_dump($parsed_articles);
+
+// echo gettype($parsed_articles);
+$threeArticles = $parsed_articles['articles'];
+$threeArticle = array_slice($threeArticles, 0, 1);
+//  var_dump($threeArticles);
+//  foreach($threeArticle as $element) {
+//     echo $element;
+//  };
+
+$twoArticles = $parsed_articles['articles'];
```

Comment:
duplicate content

Reply:
it's not a duplicate content,  i use this like that because otherwise i cant get my  lonely article in big news and the two news of the middle bloc news.